### PR TITLE
Add API free_floatings in navitia

### PR DESCRIPTION
**Pull request by kadhikari**
For details : https://jira.kisio.org/browse/NAVP-1705

### CODE REVIEW

**Review by benoit-bst — APPROVED** [review]

File: type.proto [hunk 1]
```diff
@@ -73,6 +73,32 @@ enum ResponseStatus{
...
+    SCOOTER = 2;
+    MOTORSCOOTER = 3;
+    STATION = 4;
+    CAR = 5;
+    OTHER = 6;
+}
+
+enum FreeFloatingAttributeType {
+    GEARS = 1;
+    ELECTRIC = 2;
```
Comment (pbench) [inline type.proto:87]: You have `ELECTRIC` in both `FreeFloatingAttributeType` and `FreeFloatingPropulsionType`
Is it what you want ?

**Review by pbench — APPROVED** [review]

File: type.proto (hunk 1 above)
Comment (kadhikari) [reply type.proto:87]: Yes. These are two different things as described in https://flow.fluctuo.com/api/Enum/PropulsionType and  https://flow.fluctuo.com/api/Enum/VehicleAttribute

Comment (pbench) [reply type.proto:87]: :+1:

**Review by xlqian — APPROVED** [review]

File: type.proto [hunk 2]
```diff
@@ -73,6 +73,32 @@ enum ResponseStatus{
     active_disruption = 10;
 }
 
+enum FreeFloatingType {
+    BIKE = 1;
+    SCOOTER = 2;
+    MOTORSCOOTER = 3;
+    STATION = 4;
+    CAR = 5;
+    OTHER = 6;
```
Comment (nonifier) [inline type.proto:82]: We've always set enums' fields as lower case. 
Please, follow the same conventions as the one already defined in the current file. This is clean-code #1 rule ! 
https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29#general-rules

**Review by nonifier — CHANGES_REQUESTED** [review]

**Comment by kadhikari** [comment]
Finally we don't need any proto for the new api /freefloatings_nearby as an external service (Forseti) is used instead of kraken.

**Pull request closed without merging**